src/test/java/com/plaid/client/integration/AssetReportGetTest.java

```diff
@@ -52,6 +51,7 @@ public void testAssetReportGetSuccess() throws Exception {
    */
   public static Response<AssetReportGetResponse> waitTillReady(
           PlaidClient client, String assetReportToken) throws Exception {
+    String PRODUCT_NOT_READY = "PRODUCT_NOT_READY";
```

Comment:
private final static

Reply:
It's in the function- does it make more sense to bring it out to the class level?

Reply:
yea, it's a constant. constants should be class level.

you can imagine if we want to make other utility methods that want to wait until product is ready